spec/factories.rb

```diff
@@ -32,7 +32,7 @@
     status 0
     final_date '2017/01/23'
 
-    factory :demo_night_with_projects do
+      factory :demo_night_with_projects do
```

Comment:
@notmarkmiranda Did you mean to commit this? Seems like the spacing is better without the extra.